app/src/main/java/com/nervousfish/nervousfish/activities/SettingsActivity.java

```diff
@@ -219,7 +227,7 @@ public void onCreate(final Bundle savedInstanceState) {
             // to their values. When their values change, their summaries are
             // updated to reflect the new value, per the Android Design
             // guidelines.
-            bindPreferenceSummaryToValue(findPreference("choose_verification_use_every_time"));
+            bindPreferenceSummaryToValue(findPreference("choose_verification_method_every_time"));
```

Comment:
Could you use either a `ConstantKeywords` or a `this.getString(R.strings.id)` here?